src/app/series/directives/series-feeds.component.html

```diff
@@ -19,7 +19,7 @@ <h2>No feeds found! Please contact support</h2>
           <strong *ngIf="!f.isDefault">{{ f.title || '(New Feed)' }}</strong>
           <div *ngIf="!f.isNew" class="feed-url">
             {{ f.publishedFeedUrl() }}
-            <a [href]="f.publishedFeedUrl()" target="_blank" rel="noopener" class="external-icon"></a>
+            <a [href]="f.publishedFeedUrl(true)" target="_blank" rel="noopener" class="external-icon"></a>
```

Comment:
Got some feedback that this link was confusing for private feeds.  Since it didn't include any `?auth=` token, you would open this link and get a 403 error.

This keeps the displayed value the same (no token) ... but the actual href points to your _last_ `?auth=` token if it's a private feed.